tests/unit/Unicon.spec.js

```diff
@@ -65,4 +65,28 @@ describe('Unicon', () => {
     svg.trigger('click')
     expect(wrapper.emitted().click).toBeTruthy()
   })
+  it('hover fill is working', () => {
+    const wrapper = shallowMount(Unicon, {
+      propsData: {
```

Comment:
You could simply extend `baseProps` rather than adding repeated code

Reply:
Fixed